### PR TITLE
Fixes in get started documentation

### PR DESCRIPTION
**Pull request by sukanyaparashar**
This PR changes some links and removes outdated documentation (truffle documentation since it is no longer maintained).

### CODE REVIEW

File: docs/developing/get-started.md [hunk 1]
```diff
@@ -139,13 +96,15 @@ etherscan: {
...
 ```
 
-Follow this GitHub tutorial for contract verification [github.com/neonscan/hardhat-integrate](https://github.com/neonscan/hardhat-integrate).
+Follow this GitHub tutorial for contract verification [Hardhat Github Tutorial](https://github.com/neonlabsorg/neon-tutorials/tree/main/hardhat).
 
 ## Gotchas
 
+### Reaching account limits
+
 You may encounter an error `The transaction requires too lot of accounts`, i.e. that the transaction has exceeded the account limit. This error refers to the fact that the transaction on Solana does not allow the number of accounts to be more than 64. If you do need more than 64 accounts, the smart contract function throwing the error needs to be optimized.
```
Comment (m4sterbunny) [inline docs/developing/get-started.md:108]: @sukanyaparashar Please can you trigger this error and see if the team have improved the English in the error message yet, I did ask for an update >> IF so, can you grab the text and replace?

**Comment by m4sterbunny** [comment]
I am happy with dropping Truffle, I already removed it from the home page due to sunsetting. @YYYurchenko was keen to keep it -- but this only removes it from the Get Started, and it still retains its own support page, so IMO we could clean out this item too.

File: docs/developing/get-started.md (hunk 1 above)
Comment (sukanyaparashar) [reply docs/developing/get-started.md:108]: Let me see if I can reproduce this error in some example.

**Comment by sukanyaparashar** [comment]
@m4sterbunny , I had the same suggestion in my mind to keep the support, but I think there shouldn't be any stale references in our documentation. Because ConsenSys has given the support for migrating the projects from Truffle to Hardhat.

**Comment by m4sterbunny** [comment]
OK, I guess @YYYurchenko needs to make the call -- do we clean out all references on Dec 20th: "Starting on December 20, 2023, Truffle and Ganache codebases will remain available as public archives. "

**Comment by sukanyaparashar** [comment]
@m4sterbunny , I have restored `Truffle` but i forced pushed your commit branch to mine by mistake. Please have a look at it when you review this again.

**Review by m4sterbunny — APPROVED** [review]

**Pull request merged**